test/parallel/test-crypto-keygen.js

```diff
@@ -182,7 +182,7 @@ function convertDERToPEM(label, der) {
 {
   // Test async DSA key generation.
   generateKeyPair('dsa', {
-    modulusLength: 256,
+    modulusLength: 512,
```

Comment:
Looks good to me.  I'd also be happy with this: 

```suggestion
    modulusLength: common.hasFipsCrypto ? 512 : 256,
```

Maybe even include a comment above it that explains why FIPS needs the higher setting.

Reply:
IIUC, openssl just internally increases 256 to 512 (when not FIPs), so always using 512 is no slower and more clear.